app/controllers/HistoricDateRequestPageController.scala

```diff
@@ -119,11 +128,11 @@ class HistoricDateRequestPageController @Inject()(override val messagesApi: Mess
       case (HistoricDates(start, end), v) if Period.between(
         start, end).toTotalMonths >= maximumNumberOfMonths => Some(form.withError("end",
 
-          if (v == C79Certificate) {
-            "cf.historic.document.request.form.error.date-range-too-wide.c79"
-          } else {
-            "cf.historic.document.request.form.error.date-range-too-wide"
-          }).fill(dates))
+        if (v == C79Certificate) {
+          "cf.historic.document.request.form.error.date-range-too-wide.c79"
+        } else {
+          "cf.historic.document.request.form.error.date-range-too-wide"
```

Comment:
Can we amend existing unit tests to test this case.

"cf.historic.document.request.form.error.date-range-too-wide"